pkg/webhooks/validation/server/server.go

```diff
@@ -101,13 +101,13 @@ type Options struct {
 }
 
 // String produces a stringified version of the arguments for debugging.
-func (p *Options) String() string {
+func (o Options) String() string {
 	buf := &bytes.Buffer{}
 
-	fmt.Fprintf(buf, "DomainSuffix: %s\n", p.DomainSuffix)
-	fmt.Fprintf(buf, "Port: %d\n", p.Port)
-	fmt.Fprintf(buf, "CertFile: %s\n", p.CertFile)
-	fmt.Fprintf(buf, "KeyFile: %s\n", p.KeyFile)
+	fmt.Fprintf(buf, "DomainSuffix: %s\n", o.DomainSuffix)
```

Comment:
nit: not sure if you use goland, but it always flags warnings calls to `Fprintf`, since we never use the return values.  Mind just adding `_, _ = Fprintf...`? Bonus points for switching to Goland :)